src/eredis_cluster_monitor.erl

```diff
@@ -331,6 +331,9 @@ get_cluster_info_from_connection(Connection, Query, FailFn, Node) ->
             {ok, ClusterInfo};
         {error, <<"ERR unknown command 'CLUSTER'">>} ->
             {ok, FailFn(Node)};
```

Comment:
Remove these lines, which seem wrong when checking the Redis 7,6,5 code. Your added comment below can be removed as well then.